### PR TITLE
Generate constructor should not be offered on simple names.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/9575

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-ide

**Comment by balajikris** [comment]
:+1:

**Comment by CyrusNajmabadi** [comment]
retest this please

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Comment by CyrusNajmabadi** [comment]
Ping @dotnet/roslyn-ide

**Comment by jmarolf** [comment]
retest vsi please

**Comment by jmarolf** [comment]
:+1:

File: src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateConstructor/GenerateConstructorTests.vb [hunk 1]
```diff
@@ -598,5 +598,19 @@ Class X
...
+        Public Async Function TestMissingOnMethodCall() As Task
+            Await TestMissingAsync(
+"class C
+    public sub new(int arg)
+    end sub
+
+    public function M(s as string, i as integer, b as boolean) as boolean
+        return [|M|](i, b)
+    end function
+end class")
```
Comment (DustinCampbell) [inline src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateConstructor/GenerateConstructorTests.vb]: Shift key causing you trouble today?

**Comment by DustinCampbell** [comment]
:+1:

File: src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateConstructor/GenerateConstructorTests.vb (hunk 1 above)
Comment (CyrusNajmabadi) [inline src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateConstructor/GenerateConstructorTests.vb]: I always write VB code in lowercase.  It's just line commit fixes things up for me. :)

**Comment by Pilchie** [comment]
Late, but :+1:

**Pull request merged**